tajo-catalog/tajo-catalog-server/src/main/java/org/apache/tajo/catalog/CatalogServer.java

```diff
@@ -304,9 +295,25 @@ public GetTablespaceListResponse getAllTablespaces(RpcController controller, Nul
         throws ServiceException {
       rlock.lock();
       try {
+
+        // retrieves tablespaces from catalog store
+        final List<TablespaceProto> tableSpaces = Lists.newArrayList(store.getTablespaces());
+
+        // retrieves tablespaces from linked meta data
+        tableSpaces.addAll(Collections2.transform(linkedMetadataManager.getTablespaces(),
```

Comment:
Some functional methods like transform and filter are in guava Collections2 class.